textproc/lilypond-devel/Portfile

```diff
@@ -2,10 +2,11 @@
 
 PortSystem          1.0
 PortGroup           compiler_blacklist_versions 1.0
+PortGroup           cxx11 1.1
 
 name                lilypond-devel
 set my_name         lilypond
-version             2.19.80
+version             2.19.82
 revision            4
```

Comment:
Remove the `revision` line because it starts at 0 when increasing the version and that is the default value.

Reply:
Ah, this is something not documented in https://guide.macports.org/#reference.phases!  Given that @nerdling has already merged my changes I ask you to directly fix in the git repository.

Reply:
The whole point of the revision is to trigger a rebuild for users when the version stays the same. It cannot be changed now. The next time the version is updated, remember to remove the revision line.

Reply:
The revision is documented here:
https://guide.macports.org/#reference.keywords

Reply:
OK, thanks. I'm blind sometimes...